### ISSUE TITLE
Support removing all content from an item via upload CSV

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#3181]
To fully replace XML editing, it must be possible to remove all content from a druid. The simplest way to do this would be to make the structural metadata CSV support uploading a file that consists of just the header row.

The use cases for removing all content are the decommissioning and expunging procedures. Decommissioning requires removing all files from the current version of an item, leaving them only in preservation storage. Expunging requires following the decommissioning process, followed by removing the files from the Moab.

The current procedure for decommissioning involves either deleting all resources from the contentMetadata or posting an empty contentMetadata to Fedora (if removing a batch). It is possible to accession an empty folder from Preassembly as a workaround but it would be simpler to support removing all content using the CSV within Argo.

**Comment by justinlittman** [comment]
Would it be easier to have a "remove all content", "decommission", or "expunge" button?

**Comment by andrewjbtw** [comment]
It would be extremely helpful to have buttons for decommission and expunge but both of those involve much more than the contentMetadata, so there would still be a need for a separate "remove all content" option. I ticketed decommission last year but backlogged it when I worked out how to decommission without editing XML: https://github.com/sul-dlss/argo/issues/2393

For just removing content, I'm not sure having a button would be much easier from a user perspective than being able to upload a blank CSV. We could reuse the existing interfaces with the CSV.
